admin-manual/customization/theming.rst

```diff
@@ -35,21 +36,136 @@ Customize how an AtoM theme looks
+later following `one of their guides <http://symfony.com/legacy/doc/gentle-introduction/1_4/en/17-Extending-Symfony#chapter_17_plug_ins>`_.
+
+arDominionPlugin is the default theme, i.e. the theme that will be used in a
+fresh installation. arArchivesCanadaPlugin was developed as an extension of the
+former and the following instructions will show you how to create your custom
+theme as we did with arArchivesCanadaPlugin.
+
+Assuming that you already have AtoM installed in your development environment
+(you can use our :ref:`Vagrant box <installation-vagrant>`), let's start
+building the plugin structure from the command line. Our theme is going to be
+called Corcovado (arCorcovadoPlugin). We are going to track its contents with
+git and publish them in a remote repository hosted by GitHub so we can enable
+others to contribute in the development. The repository is open source so you
+can use it for your own reference, see https://github.com/artefactual-labs/atom-theme-corcovado.
+You can also `create your own repository <https://help.github.com/articles/create-a-repo/>`_.
+
+Let's begin to do some real work:
+
+.. code-block:: bash
+
+   $ cd ~/atom/plugins
+   $ mkdir arCorcovadoPlugin
+   $ git init
+   $ echo "# Corcovado theme" > README.md
+   $ git add README.md
+   $ git commit -m "Initial commit"
+   $ git remote add origin git@github.com:artefactual-labs/atom-theme-corcovado.git
+   $ git push
+
+We've created an empty directory where our plugin is going to be contained,
+made a git repository of it, track our first file and publish our work to
+GitHub! However, our plugin does not meet the needed requirements for AtoM to
+recognize it and allow us to enable it. Let's make that happen:
+
+.. code-block:: bash
+
+   $ cd ~/atom/plugins/arCorcovadoPlugin
+   $ mkdir config
+   $ cd config
+
+Create a new file :file:`arCorcovadoPluginConfiguration.class.php` with the
+following contents:
+
+.. code-block:: php
+
+   <?php
+
+   class arCorcovadoPluginConfiguration extends sfPluginConfiguration
+   {
+     public static
+       $summary = 'Theme plugin, extension of arCorcovadoPlugin.',
+       $version = '0.0.1';
+
+     public function contextLoadFactories(sfEvent $event)
+     {
+       $context = $event->getSubject();
+       $context->response->addStylesheet('/plugins/arCorcovadoPlugin/css/min.css', 'last', array('media' => 'all'));
+     }
+
+     public function initialize()
+     {
+       $this->dispatcher->connect('context.load_factories', array($this, 'contextLoadFactories'));
+
+       $decoratorDirs = sfConfig::get('sf_decorator_dirs');
+       $decoratorDirs[] = $this->rootDir.'/templates';
+       sfConfig::set('sf_decorator_dirs', $decoratorDirs);
+
+       $moduleDirs = sfConfig::get('sf_module_dirs');
+       $moduleDirs[$this->rootDir.'/modules'] = false;
+       sfConfig::set('sf_module_dirs', $moduleDirs);
+     }
+   }
+
+You may also have to clear the Symfony cache, depending on the configuration of
```

Comment:
link "clear the Symfony cache" to https://www.accesstomemory.org/docs/2.3/admin-manual/maintenance/clear-cache/ ?